docs/design/2023-02-23-cloud-native-architecture.md

```diff
@@ -85,9 +76,9 @@ Based on the above analysis, it is impossible to find a perfect shared storage t
     * Including various files written to disk in the DeltaTree storage engine
     * S3 provides high durability and high availability. So this part of the data should not be sensitive to storage latency
     * RN have to request WN for other data during queries.
-2. IO latency sensitive data is directly written to Write Node's local disk and uploaded to S3 regularly. It is equivalent to using local disk as buffer
+2. IO latency sensitive data is directly written to Write Node's local disk and uploaded to S3 peroidically. I.e. it uses local disk as buffer
     * About durability - if data from local disk is lost, recover from TiKV. Equivalent to relying on upstream TiKV to provide high durability.
-    * About availability - Data needs to be resynchronized after restart. And the time for service recovery is affected by the delay of synchronized data. Since we upload data regularly, the actual data to be resynchronized is incremental data relative to the last upload point, and the total amount should be small, so it is considered acceptable.
+    * About availability - Data needs to be resynchronized after restart. And the time for service recovery is affected by the delay of synchronized data. Since we upload data peroidically (for example 30s), the actual data to be resynchronized is incremental data relative to the last upload point. The total amount of data should be small, so it should not take very long time. It means the available duration should not be a big issue.
```

Comment:
```suggestion
    * About availability - Data needs to be resynchronized after restart. And the time for service recovery is affected by the delay of synchronized data. Since we upload data periodically (for example 30s), the actual data to be resynchronized is incremental data relative to the last upload point. The total amount of data should be small, so it should not take very long time. It means the available duration should not be a big issue.
```